templates/base-conditional.html

```diff
@@ -109,6 +109,7 @@
     {% include "bootstrap-helpers.html" %}
 
     {% include "portfolios/select-portfolio-modal.html" %}
+    {% include "portfolios/manage_portfolio_modal.html" %}
```

Comment:
Instead of including this in a global template, perhaps this can be included only in the templates where it's used, like the portfolios_table.html template.

Reply:
I see now that this is consistent with the "select portfolio" modal. I'm not a fan of that logic in general, but maybe this should all be refactored later, if not done now.

Reply:
I concur.

Reply:
@davidpofo Even if the entire issue isn't resolved here, this PR would be a good place to move these specific new include statements into somewhere like the portfolios_table.html template, or wherever else they're used.